runtime/src/main/java/org/corfudb/runtime/LogReplicationUtils.java

```diff
@@ -0,0 +1,14 @@
+package org.corfudb.runtime;
+
+
+/**
+ * LogReplication code resides in the infrastructure package.  Adding a dependency from this package(runtime) to
+ * infrastructure introduces a circular dependency.  This class defines LR-specific constants and utility methods
+ * required in runtime.  Note that these methods are unique and not duplicated from infrastructure.
+ */
+public class LogReplicationUtils {
+
+    public static final String LR_STATUS_STREAM_TAG = "lr_status";
+
+    public static final String REPLICATION_STATUS_TABLE = "LogReplicationStatus";
```

Comment:
I've been thinking we need a utils class as well, do you think it will be good to move all of our static assets (stream tags, table names, merge-only stream IDs, etc) to this class? maybe we can add this item to our backlog if not in this pr?

Reply:
this class is for those constants and utility methods which are needed in the runtime package to avoid a circular dependency between the runtime and infrastructure(see class-level comment).  If there are other such examples, we can surely move them here.

I agree, we can create a backlog item.